frontend/src/components/HomePage/HomePage.tsx

```diff
@@ -50,6 +40,11 @@ function HomePage() {
         style={style}
         imageSource={panoramaImage}
         config={config}
+        onPanoramaLoaded={() => {
+          setTimeout(() => {
```

Comment:
is this timer necessary?

Reply:
It calls itself slightly before it actually fully loads so you still see the loading bar for some reason?

Reply:
interesting, thanks for clarifying - maybe just add a comment in the code that mentions this then ill approve

Reply:
done ⭐